src/server/streamableHttp.ts

```diff
@@ -35,12 +35,12 @@ export interface StreamableHTTPServerTransportOptions {
  * ```typescript
  * // Stateful mode - server sets the session ID
  * const statefulTransport = new StreamableHTTPServerTransport({
- *  sessionId: randomUUID(),
+ *   sessionIdGenerator: () => randomUUID(),
  * });
  * 
  * // Stateless mode - explicitly set session ID to undefined
  * const statelessTransport = new StreamableHTTPServerTransport({
- *    sessionId: undefined,
+ *   sessionIdGenerator: () => undefined,
```

Comment:
`sessionIdGenerator` changed

```suggestion
 *   sessionIdGenerator: undefined,
```